templates/tfengine/components/devops/main.tf

```diff
@@ -163,13 +165,13 @@ resource "time_sleep" "admins_wait" {
 # Admin permission at {{.parent_type}} level.
```

Comment:
Unfortunately I cannot write terraform vars here like

```suggestion
# Admin permission at ${var.parent_type} level.
```

So I decided not to include parent_type on `variables.tf`, potentially causing that in a variable change, this comment won't be accurate.

This case is less problematic than the previous one within the `terraform` block, but I just wanted to point it out

Reply:
You're right. But even if it's possible. I don't think it worth to have the possibility of an inconsistency. I'll keep it as is